vos/vos/vos.py

```diff
@@ -1666,9 +1670,11 @@ def copy(self, source, destination, send_md5=False, disposition=False):
                             destination)
                         logger.debug(
                             "{0} {1}".format(source_md5, destination_md5))
-                        assert destination_md5 == source_md5, \
-                            ('Source and destination md5 do not match: '
-                             '{} vs. {}').format(source_md5, destination_md5)
+                        if not fhead and (destination_md5 != source_md5):
```

Comment:
This change could be removed if check_md5 was not set to True during the MD5 Check section up-above.  

Generally copy is a bit ugly these days.